libs/ui-lib/lib/ocm/components/clusterWizard/wizardTransition.ts

```diff
@@ -245,6 +247,19 @@ const reviewStepValidationsMap: WizardStepValidationMap = {
   softValidationIds: [],
 };
 
+const kubeconfigValidationMap: WizardStepValidationMap = {
```

Comment:
I dont think we need any validations on this step
```suggestion
const kubeconfigValidationMap = buildEmptyValidationsMap();
```